docs/platforms/dotnet/guides/aspnet/index.mdx

```diff
@@ -34,7 +34,7 @@ as well as your logs as breadcrumbs. The logging integrations also capture event
 You configure the SDK in the `Global.asax.cs`:
 
 
-```csharp {"onboardingOptions": {"performance": "22-25, 35-44"}}
```

Comment:
I love the refactoring-friendliness of this change!

Would you mind applying this improvement also to
- Common .NET
https://github.com/getsentry/sentry-docs/blob/6f44215f7eb215d68024537b3e7cdf83ed709f8c/docs/platforms/dotnet/index.mdx?plain=1#L84
- Xamarin
https://github.com/getsentry/sentry-docs/blob/0c427cb216682ffb4f943d402cbb37c473492d97/docs/platforms/dotnet/guides/xamarin/index.mdx?plain=1#L29
?